mathics/builtin/specialfns/elliptic.py

```diff
@@ -24,8 +24,8 @@ class EllipticE(SympyFunction):
       <dt>'EllipticE[$m$]'
       <dd>computes the complete elliptic integral $E$($m$).
 
-      <dt>'EllipticE[ϕ|$m$]'
-      <dd>computes the complete elliptic integral of the second kind $E$($m$|$ϕ$).
+      <dt>'EllipticE[phi|$m$]'
+      <dd>computes the complete elliptic integral of the second kind $E$($m$|phi).
```

Comment:
`$phi$`?

Reply:
Ah - ok. But in my mind what is really needed is this junk to be able to understand Math mode. 

The unicode symbols were removed when I notice that we can't handle this in building the LaTeX doc. Sigh.